design-tokens/tokens/component.default.json

```diff
@@ -6722,6 +6750,100 @@
         }
       }
     },
+    "units":{
+      "enabled": {
+        "textColor": {
+          "$type": "color",
+          "$value": "{color.text.weak}",
+          "$description": "",
+          "$extensions": {
+            "com.figma": {
+              "hiddenFromPublishing": false,
+              "scopes": [
+                "TEXT_FILL"
+              ],
+              "codeSyntax": {}
+            }
+          }
+        },
+        "borderColor": {
```

Comment:
I wonder since "units" is just a text element, it feels like we can remove "borderColor" and "background" from each of these. What do you think?

Reply:
yeah I also wasnt sure about since "borderColor" and "background" were in the default section

Reply:
took out